package.json

```diff
@@ -27,20 +27,20 @@
     "start": "tsdx watch --format esm --target node",
     "build": "tsdx build --format esm --target node",
     "test": "tsdx test",
-    "test-local": "rm -rf ./__tests__/fixtures/__temp && yarn test",
+    "test-local": "rm -rf ./__tests__/fixtures/__temp && pnpm test",
     "lint": "eslint . --max-warnings 0",
-    "prepare": "yarn build",
+    "prepare": "pnpm build",
     "size": "size-limit",
     "analyze": "size-limit --why",
     "build-docker-ts": "tsc --project docker/tsconfig.build.json",
     "postinstall": "patch-package"
   },
-  "peerDependencies": {},
   "devDependencies": {
     "@babel/eslint-parser": "7.21.3",
     "@babel/plugin-transform-modules-commonjs": "7.21.2",
     "@next/eslint-plugin-next": "13.3.0",
```

Comment:
- probably we can get rid of a lot of these devDependencies as well, especially the ESLint ones
- and you can switch from `@upleveled/eslint-config-upleveled` to `eslint-config-upleveled`, which is the new package name